Header header
float64 altitude
int32 unavailability
